src/shared/dissect-image.c

```diff
@@ -204,6 +204,7 @@ static int make_partition_devname(
 
 int dissect_image(
                 int fd,
+                const char *original_filename,
```

Comment:
this can be a fully path, no? hence name `original_path`?

usually if we call something "filename" we expect it to not include slashe,s i.e. really be just a filename, not a path